Lib9c/Helper/CrystalCalculator.cs

```diff
@@ -58,5 +58,32 @@ bool enhancementFailed
             var extra = crystal.DivRem(100, out _) * multiplierRow.Multiplier;
             return crystal + extra;
         }
+
+        /// <param name="materials"> Key : id of material, Value : count of material </param>
+        public static FungibleAssetValue CalculateMaterialCost(
```

Comment:
Is this used method?

Reply:
It is used in 9c client. Is it not okay to keep unused method in lib9c?

Reply:
If it's only used on client, it might be better to move it to the client side.